lib/classes/organisations_user_builder.rb

```diff
@@ -48,10 +50,14 @@ def organisation
   end
 
   def update_user
+    update_user_address if @user_address_attributes['district_id'] && @district
     @user.roles << charity_role unless @user.roles.include?(charity_role)
     @user.update(@user_attributes)
   end
 
+  def update_user_address
+    @user.address.update(district_id: @user_address_attributes['district_id'])
```

Comment:
We need to verify scenario where  user do not have existing address record.

Reply:
Hi @namrataukirde , Address gets created as soon as user is created, but however, i have added an edge case if user doesn't have address, in that case we'll create the address with the same params, coming from Client.